api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1232 @@
+        backing messaging implementation needed for the reception and persistency of the Events. Considering
+        that at this time only Kafka is used, this corresponds to the creation of a Topic. If so desired,
+        clients can interact directly with the topic using the low level API (for existing restrictions see
+        the corresponding methods on the topic-api).
+
+        The fields validation-strategies, enrichment-strategies and partition-resolution-strategy have all an
+        effect on the incoming Event of this EventType. For its impacts on the reception of events please
+        consult the Event submission API methods.
+
+        * Validation strategies define an array of `EventValidationStrategy` to be evaluated on reception of
+        an `Event` of this `EventType`. Details of usage can be found in an external document (TBD link to
+        document).
+        **Notes**: Some EventValidationStrategies might support additional parameters. The behaviour is
+        specific to each strategy and one is suggested to consult its `doc` field for detailes of operation.
+        The default `SchemaValidationStrategy` supports passing fields to be ignored if the qualifier matches
+        the input.
+
+        * TDB Enrichment strategy
+
+        * TDB Partition resolution strategy
+
+        * The schema of an `EventType` is defined as an `EventTypeSchema`. Currently only json-schema is
+        supported.
+
+        Following violations are enforced. Not meeting them will fail the request with a 422 Unprocessable
+        entity (details are provided in the Problem object):
+
+        * Event name on creation not begin unique.
+
+        * Using `EventTypeSchema.type` other than json-schema or passing a `EventTypeSchema.schema`
+        that is invalid with respect to the schema's type.
+
+        * Referring any of Validation, Enrichment or Partitioning strategies that does not exist or whose
+        parameterization is deemed invalid.
+
+        Nakadi MIGHT impose necessary schema, validation and enrichment minimal configurations that MUST be
+        followed by all EventTypes (examples include: validation rules to match the schema; enriching every
+        Event with the reception date-type; adhering to a set of schema fields that are mandatory for all
+        EventTypes). **The mechanism to set and inspect such rules is not defined at this time and might not
+        be exposed in the API.**
+
+      parameters:
+        - name: event-type
+          in: body
+          description: EventType to be created
+          schema:
+            type: object
+            $ref: '#/definitions/EventType'
+          required: true
+      responses:
+        '200':
+          description: Ok
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '422':
```

Comment:
The request should be rejected, if the event type resource instance already exists, shouldn't it?
Because than I have to use PUT.
It would be adequate to return 422 here and should be mentioned in the spec.